blaze-server/src/test/scala/org/http4s/server/blaze/Http1ServerStageSpec.scala

```diff
@@ -5,13 +5,15 @@ import cats.effect._
 import cats.implicits._
 import java.nio.ByteBuffer
 import java.nio.charset.StandardCharsets
+
```

Comment:
Imports should be sorted alphabetically without any blank lines.

Reply:
got it
btw do you know why build fails with my `Link.scala` with the format issues i guess?
locally it passed with the `test:scalafmt`

Reply:
`test:scalafmt` only formats test sources. You need to also run `scalafmt` to format main sources.

You can run `sbt ci` to do the same as what ci runs. It takes a while though.